src/shared/components/post/post-form.tsx

```diff
@@ -364,6 +380,7 @@ export class PostForm extends Component<PostFormProps, PostFormState> {
           <div className="col-sm-10">
             <input
               type="url"
+              placeholder={I18NextService.i18n.t("optional")}
```

Comment:
Also added optional to a lot of these fields, to make it more clear, especially since these new ones aren't required.